### DISCUSSION TITLE
Prioritization

### DISCUSSION THREAD

**Discussion by Ahmad-2213** [XTLS/Xray-core#3803] [Q&A]
I've got some vmess configuration in vnext section , I want to prioritize which configuration xray has to connect to and set a health-check to see if the prioritized configuration works or not? if so , then it will continue connecting to the prioritized config, if the config fails , then it connects to second server and so on and it would do the same until the first config come alive and connect to it.
does xray support such Prioritization?

**Comment by mmmray** [comment]
you are looking for observatory/burstObservatory. but such a function is implemented in a more user-friendly way in hiddify.

**Comment by Ahmad-2213** [comment]
you mean xray doesn't support Prioritization ?
will it be added in later updates?
